### DISCUSSION TITLE
`FindMany` with nested relations

### DISCUSSION THREAD

**Discussion by JefferyHus** [prisma/prisma#21396] [Q&A] [answered]
### Bug description

I am trying to find multiple rows using nested relations, but it throws multiple errors `Unknown arg 'is' in where.service.is for type ServiceWhereInput. Did you mean 'id'?`

This is the where clause I am trying:

```javascript
{
      where: {
        service: {
          is: {
            user: {
              is: {
                id: agentId,
              }
            }
          }
        },
      },
}
```

When I get rid of the `user` part, everything works like it is supposed to. Otherwise it throws the error above.

### How to reproduce

- Create 3 types where each one has a relation with the previous.
- Then go to the last model in that relation chain and try to `findMany` using the where clause.

### Expected behavior

It should return a list of `servicePlanning` object.

### Prisma information

```prisma
model users {
  id                  Int              @id @default(autoincrement())

  Service             Service[]

  @@map("users")
}

model Service {
  id                Int       @id @default(autoincrement())
  userId         Int

  user            users             @relation(fields: [userId], references: [id])
  ServicePlanning ServicePlanning[]

  @@map("service")
}

model ServicePlanning {
  id        Int                 @id @default(autoincrement())
  serviceId Int

  service   Service             @relation(fields: [serviceId], references: [id])

  @@map("service_planning")
}

enum ServiceAvailability {
  available
  unavailable
  occupied
}
```

```ts
    prisma.servicePlanning.findMany({
      where: {
        service: {
          is: {
            user: {
              is: {
                id: agentId,
              }
            }
          }
        },
      },
      include: {
        service: true,
      },
    });
```

### Environment & setup

- OS: macOS
- Database: MySQL
- Node.js version: v18.16.0

### Prisma Version

```
^4.13.0
```

**Comment by JefferyHus** [comment, marked as answer]
I found the issue, it was that the value of `agentId` was wrongly formatted to a string instead of a number, but then the error still does not make sense, the error should be type related and not a usability error

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
